grant-project/zcash-grant-system
# Email subscription fixes

Closes #5, makes some other improvements to emails.

### What This Does

* Fix the links in the footers of emails to function correctly.
* Looks up user email settings by to email address, to ensure emails don't get sent when undesired.
* Fixes some copy.

### Steps to Test

* Get yourself sent an email (e.g. change password)
* Click unsubscribe link, confirm it sends you to the right path (Requires properly configured .env to go to right URL.)
